content/millennium/r4/financial/financial-transaction.md

```diff
@@ -0,0 +1,206 @@
+
+ ID                                      | Value\[x] Type      | Description
+-----------------------------------------|---------------------|--------------------------------------------------------------------------
+ `financial-transaction-account-number`  | [`string`]          | A value associated to the specific payment method usually represented as the last four digits of a credit card, the check number, the EFT number or Lockbox number.
+ `financial-transaction-alias`           | [`string`]          | Client defined value to represent the combination of the type, subtype, and reason describing the financial transaction.
+ `financial-transaction-allocation`      | [`Extension`]       | Defines how the payment or adjustment is to be allocated across other resources.
+ `financial-transaction-amount`          | [`Money`]           | The total amount of the financial transaction.
+ `financial-transaction-card-brand`      | [`string`]          | Identifies the brand of credit card when credit card is used as a payment method.
+ `financial-transaction-date`            | [`date`]            | Represents the expiration date if method is card, check date if method is check, EFT date if method is EFT or Lockbox date if method is lockbox.
+ `financial-transaction-location`        | [`string`]          | Client configured value representing the location or workflow that the payment was received in.
+ `financial-transaction-method`          | [`string`]          | Describes the method of payment for the financial transaction.
+ `financial-transaction-tendered-amount` | [`Money`]           | The amount of cash originally tendered for payment. This value should be greater than or equal to the amount of the cash payment.
+ `financial-transaction-type`            | [`CodeableConcept`] | The classification of the transaction.
+
+## Create
+
+Create a FinancialTransaction.
+
+    POST /Basic    
+
+_Implementation Notes_
+
+* Only the body fields mentioned below are supported.
+* When integrating your application with a client's production environment, the client will have to provide the appropriate financial-transaction-alias values to send with the financial transaction.
+* The financial-transaction-allocation.target field must reference an Encounter associated with an Account.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token&gt;', 'Content-Type': 'application/fhir+json'} %>
+
+### Body Fields
+
+<%= definition_table(:financial_transaction, :create, :r4) %>
+
+### Example
+
+#### Request
+
+    POST /Basic
+
+#### Body
+
+  <%= json(:R4_FINANCIAL_TRANSACTION_CASH) %>
+
+#### Response
+
+<%= headers status: 201 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: application/fhir+json
+Date: Mon, 04 Nov 2019 16:23:57 GMT
+Etag: W/""
```

Comment:
Let's make all the Etags have: W/"0"

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/4adf16f12daf8a74d8b1caef1c28f22dd3d0b8fd#diff-a2a1182a3c7ddb3de2fa0458e90ddf0cR93

Reply:
Updated all e tags
https://github.com/cerner/fhir.cerner.com/pull/297/commits/419ca8f22c096f653df2e12cfffd8fa90b9b87d5#diff-a2a1182a3c7ddb3de2fa0458e90ddf0c

Reply:
Should be 4 etag updates

Reply:
The first commit is one and the second commit has 3 updates.

Reply:
+1